app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -117,6 +120,12 @@ def check_enrollment(enrollment)
     enrollment.update(status_check_attempted_at: status_check_attempted_at)
   end
 
+  def passed_with_unsupported_secondary_id_type?(enrollment, response)
+    return enrollment.capture_secondary_id_enabled &&
+           response['secondaryIdType'].present? &&
+           SUPPORTED_SECONDARY_ID_TYPES.exclude?(response['secondaryIdType'])
```

Comment:
want to make sure i'm getting this correct - if the secondaryIdType is marked as `'Visual Inspection of Name and Address on Primary ID Match'` by the po worker then this will return false, so this function is saying yes the id is in fact unsupported?

Reply:
line 126 will evaluate `true` if `response['secondaryIdType']` is **not** in `SUPPORTED_SECONDARY_ID_TYPES` (I'm using `exclude?` ([documentation](https://api.rubyonrails.org/v5.2/classes/Enumerable.html#method-i-exclude-3F)) instead of `include?`). So if that's true, then it has been passed with an unsupported secondary ID type, which means we should fail (on line 417)